collectors/python.d.plugin/logparser/logparser.chart.py

```diff
@@ -54,8 +55,8 @@ def populate_matchers(self):
         for name, pattern in self.dimensions.items():
             try:
                 matcher = matcher_factory(pattern)
-            except ValueError as error:
-                self.error("error on creating matchers : {0}".format(error))
+            except ValueError as re.error:
+                self.error("error on creating matchers : {0}".format(re.error))
```

Comment:
no no 

```python
except (ValueError, re.error) as error:
```

Reply:
this is needed because of `re.compile` is not safe, it can raise exception of type `re.error`